src/probeinterface/probe.py

```diff
@@ -15,7 +15,7 @@ class Probe:
 
     """
 
-    def __init__(self, ndim=2, si_units="um"):
+    def __init__(self, ndim=2, si_units="um", name=None, serial_number=None, model_name=None, manufacturer=None):
```

Comment:
I am not sure to like the idea of overloading the constructor with fixed annotation even if they are important.

When creating a Probe from scratch (wich a end user normally do not do) I think I prefer this:

```python
probe = Probe(ndim=2)
probe.annotate(name=name)
probe.annotate(serial_number=serial_number)
probe.annotate(model_name=model_name)
probe.annotate(manufacturer=manufacturer)
```

The discussion of which annotations are important are endless.
So lets put annotations outside the constructor.

The other solution would be to put all of then (but I do not like)
```python
probe = Probe(ndim=2, **any_annotations)
```

Reply:
or another solution would be to make `set_*` for these 4 key annotations, so that they are still annotations (i.e., saved to file too), but the have an easier API. Whaat do you think?